packages/expo-modules-core/android/src/main/java/expo/modules/adapters/react/permissions/PermissionsService.kt

```diff
@@ -162,7 +162,7 @@ open class PermissionsService(val context: Context) : InternalModule, Permission
   override fun isPermissionPresentInManifest(permission: String): Boolean {
     try {
       context.packageManager.getPackageInfo(context.packageName, PackageManager.GET_PERMISSIONS)?.run {
-        return requestedPermissions.contains(permission)
+        return requestedPermissions!!.contains(permission)
```

Comment:
❓ Question:
Isn't there a risk of crash here with the non-null assertion `!!`?
There is no catch for a `NullPointerException` below in case the value `requestedPermissions ` is `null`.

I am not Kotlin-fluent, so maybe I am missing some nuance.